go.mod

```diff
@@ -38,8 +38,8 @@ require (
 	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20 // indirect
 	github.com/containernetworking/cni v0.7.1
 	github.com/coredns/corefile-migration v1.0.2
-	github.com/coreos/etcd v3.3.15+incompatible
-	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7
+	github.com/coreos/etcd v3.3.17+incompatible
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
```

Comment:
Is there a reason for backporting this bump of `go-systemd`, or is it just a typo?

It was introduced in master here: https://github.com/kubernetes/kubernetes/pull/83527

On another note, I have had best luck with just using `hack/pin-dependency.sh` && `hack/update-vendor.sh` when backporting dependency changes into release branches, since cherry-picking can (and often will) result in some strange results.

Reply:
`hack/lint-dependencies.sh` requested I bump to this version:

```
github.com/coreos/go-systemd                                              
    pinned:    v0.0.0-20180511133405-39ca1b05acc7                                                                                                 
    preferred: v0.0.0-20190321100706-95778dfbb74e                                                    
    hack/pin-dependency.sh github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
```

Reply:
Also upgraded dbus to 2ff6f7ffd60f like was done in https://github.com/kubernetes/kubernetes/pull/83527